services/QuillLMS/app/models/schools_admins.rb

```diff
@@ -37,6 +38,7 @@ def wipe_cache
     $redis.del("#{DISTRICT_ACTIVITY_SCORES_CACHE_KEY_STEM}#{user_id}")
     $redis.del("#{DISTRICT_CONCEPT_REPORTS_CACHE_KEY_STEM}#{user_id}")
     $redis.del("#{DISTRICT_STANDARD_REPORTS_CACHE_KEY_STEM}#{user_id}")
+    $redis.del("#{FREEMIUM_DISTRICT_STANDARD_REPORTS_CACHE_KEY_STEM}#{user_id}")
```

Comment:
We should avoid using global variables (i.e `$redis`).  Instead use [delete](https://api.rubyonrails.org/v7.0.4/classes/ActiveSupport/Cache/Store.html#method-i-delete) with a Rails wrapper:

 ```
Rails.cache.delete("#{FREEMIUM_DISTRICT_STANDARD_REPORTS_CACHE_KEY_STEM}#{user_id}")
```

The added benefit to this is that we abstract out the details of what the cache is.  If at some point we used a different cache store, we only have to change the config [here](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/config/application.rb#L27) rather than replacing $redis everywhere.

Reply:
Thanks for the explanation! I'll make the changes for that here and the other places where it's used